docs/tools/swift.pod

```diff
@@ -1,30 +1,106 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift -- General-purpose programming language
```

Comment:
How's "high-performance system programming language" from the first line of the README instead?